docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Gift Cards
+---
+
+:::note
+Gift Cards are available in Saleor 3.1 (alpha stage). For production purposes, we recommend using the latest beta version of Saleor 3.0.
+:::
+
+This page contains information on how to manage gift cards by selecting a specific currency, sending them to customers, or setting expiring dates.
+
+## Introduction
+Gift cards are timeless and universal gifts. They are suitable for customers, employees, CEOs, business partners, or family members. Issuing gift cards can increase your sales and provide a better customer experience.
+
+## How to
+### How to create a gift card?
+Once you have created a product type that can be used as a gift card or voucher, you can issue a gift card. Go to the Gift Card page of the Catalog section. 
+
+![Gift cards](../screenshots/catalog-giftcards-main.png)
+
+Then click the Issue Card button on the top right corner of the page. A pop-up page will appear where you can carry out the following actions related to the gift card creation:  
+
+- Choose currency  
+
+- Enter the amount  
+
+- Send gift cards to customers  
```

Comment:
I think the important thing to note somewhere here, as it can be a bit confusing, is that the customer will have the gift card assigned to them when they use it in checkout. The gift card can be passed to other people and not necessarily used by the selected customer only. 

```suggestion
- Send gift cards to customer
```

Reply:
My bad--used a plural word.